Client/mods/deathmatch/logic/CClientGame.cpp

```diff
@@ -5492,6 +5492,11 @@ void CClientGame::ResetMapInfo()
     g_pMultiplayer->SetGrainMultiplier(eGrainMultiplierType::ALL, 1.0f);
     g_pMultiplayer->SetGrainLevel(0);
 
+    // Underwater effects
+    g_pMultiplayer->SetUnderwaterDarkness(true, 90.0f);
+    g_pMultiplayer->SetUnderwaterEffectEnabled(false);
+    g_pMultiplayer->SetUnderwaterEffectSpeed(0.0015f, 0.04f);
```

Comment:
It would've been better if you added a reset function to the multiplayer SDK instead of having these unnamed constants everywhere.